client/src/components/Popup/Popup.tsx

```diff
@@ -1,13 +1,13 @@
 import "./Popup.scss";
 import classnames from "classnames";
 
-interface PopUpProps {
+interface PopupProps {
   className?: string;
   isShown: boolean;
   handleIsShown?: (value: boolean) => void;
   children?: React.ReactNode;
 }
-const PopUp: React.FC<PopUpProps> = (props) => {
+const Popup: React.FC<PopupProps> = (props) => {
```

Comment:
Suggest renaming this component Modal, instead of changing to Popup, as that's what it is really.

Reply:
Much better name - with no ambiguities!

I've renamed all things popup -> Modal.

I also replaced 'popup' classnames in Modal.scss to 'modal' and mirrored this in the Modal component.

Could you have a look at those and make sure I have screwed up the styling?

Cheers, Paul.

Reply:
LOL - make sure I haven't messed up the styling!